docs/gettinginvolved/cluster-setup.rst

```diff
@@ -0,0 +1,200 @@
+   If this is not specified, packages will be installed from a release repository.
+   The release repository may be a stable or unstable repository, and will be selected depending on the version to be installed.
+
+.. option:: --flocker-version <version>
+
+   Specifies the version of Flocker to install from the selected repository.
+   If this is not specified (or is an empty string), the most recent version available in the repository will be installed.
+
+   .. note::
+
+      The build server merges forward before building packages, except on release branches.
+      If you want to use a branch in development, you probably only want to specify the branch.
+
+.. option:: --build-server <buildserver>
+
+   Specifies the base URL of the build server to install from.
+   This is probably only useful when testing changes to the build server.
+
+.. option:: --config-file <config-file>
+
+   Specifies a YAML configuration file that contains provider specific configuration.
+   See below for the required configuration options.
+
+.. option:: --no-keep
+
+   Destroy the cluster after creating it.
+   This option is useful for testing the tool itself.
+
+.. option:: --purpose <purpose>
+
+   Specifies a string that describes the purpose of the cluster.
+   This can be included into the metadata describing the cluster nodes and/or names of the nodes.
+
+.. option:: --number-of-nodes <number>
+
+   Specifies the number of nodes (machines) to create for use in the cluster.
+   This option is only applicable if the nodes are created dynamically.
+
+.. option:: --app-template <application-template-file>
+
+   Specifies a YAML file that describes a single application.
+   It must include a name of a Docker image to use as an application container and may include other parameters.
+
+.. option:: --apps-per-node <number>
+
+   Specifies the number of applications (containers) to start on each cluster node.
+   If this is not specified or zero, then no applications will be started.
+
+To see the supported values for each option, run:
+
+.. prompt:: bash $
+
+   admin/setup-cluster --help
+
+Application Template
+====================
+
+The configuration file given for the ``--app-template`` parameter describes a single application.
+At the very least it should specify a name of a Docker image to use for an application container.
+
+.. code-block:: yaml
+
+  image: "clusterhq/mongodb"
+  volume:
+    mountpoint: "/data/db"
+
+See :doc:`../control/cli/application-config` for more details.
+The ``--apps-per-node`` parameter specifies how many applications to start on each cluster node.
+
+.. prompt:: bash $
+
+  admin/setup-cluster \
+    --distribution centos-7 \
+    --provider rackspace \
+    --config-file $PWD/cluster.yml \
+    --number-of-nodes 2 \
+    --app-template $PWD/application.yml \
+    --apps-per-node 5
+
+Note that all application instances will have exactly the same configuration.
```

Comment:
Add "In particular, multiple containers may fail to start if they use a common host resource (e.g. host ports)."